bin/http-server

```diff
@@ -21,6 +21,7 @@ if (argv.h || argv.help) {
     '  -p           Port to use [8080]',
     '  -a           Address to use [0.0.0.0]',
     '  -d           Show directory listings [true]',
+    '  -b           Base directory to serve files from [/]',
```

Comment:
This should document using `--baseDir` as well. Same for the README.

Reply:
Looks like the other multi-word options (e.g. `--log-ip`) use kebab-case so I'll do `--base-dir` for this.